### PR TITLE
parrot: Added files

### PR DESCRIPTION
**Pull request by alr0cks**
Flip flop flip flop get my configs

### CODE REVIEW

**Comment by anunaym14** [comment]
Will this hack my laptop though? xD

File: parrot/setup_main.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+sudo apt autoremove --purge snapd apparmor ^avahi rsyslog apport ^vim
+rm -rf ~/snap
+
+#install oh-my-zsh and powerlevel10k
+chsh -s /usr/bin/zsh
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"
+git clone --depth=1 https://github.com/romkatv/powerlevel10k.git ${ZSH_CUSTOM:-$HOME/.oh-my-zsh/custom}/themes/powerlevel10k
+
+#update zshrc
+sudo cp Dotfiles/.zshrc ~/
```
Comment (akhilnarang) [inline parrot/setup_main.sh]: This should most definitely not need or use sudo

File: parrot/setup_kde.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+sudo apt install parrot-kde sddm 
+
+sudo systemctl enable sddm.service
+
+sudo cp -r -v Dotfiles/.config/* ~/.config/
+sudo cp -r -v Dotfiles/.local/* ~/.local/
```
Comment (akhilnarang) [inline parrot/setup_kde.sh]: Same, no sudo

**Review by akhilnarang — CHANGES_REQUESTED** [review]

File: parrot/setup_kde.sh (hunk 2 above)
Comment (anunaym14) [reply parrot/setup_kde.sh]: where's the fun without sudo though

File: parrot/install_nvidia_drivers_post_boot.sh [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	Device "DiscreteNvidia"
+EndSection
+EOL
+
+#Installing OpenCL driver to make your hashcat and any other GUI programs work
+sudo apt install -y ocl-icd-libopencl1 nvidia-cuda-toolkit
+
+echo "Do you want to reboot now [Y/n] ?"
+read inputs
+if [$inputs == "Y" || $inputs == "y"]; then
```
Comment (akhilnarang) [inline parrot/install_nvidia_drivers_post_boot.sh]: Also does this check work?🤔

Comment (alr0cks) [reply parrot/install_nvidia_drivers_post_boot.sh]: check and tell XD

Comment (akhilnarang) [reply parrot/install_nvidia_drivers_post_boot.sh]: Pretty sure it won't, in any case can be simplified atleast

**Review by alr0cks — COMMENTED** [review]
check now

File: parrot/install_nvidia_drivers_post_boot.sh (hunk 3 above)
Comment (gotenksIN) [reply parrot/install_nvidia_drivers_post_boot.sh]: well it worked in mine a few months ago atleast

**Comment by gotenksIN** [comment]
also y'all taking code review so seriously, considering that original code is garbage itself

File: parrot/install_nvidia_drivers_post_boot.sh (hunk 3 above)
Comment (akhilnarang) [reply parrot/install_nvidia_drivers_post_boot.sh]: `line 3: [y: command not found`

Comment (akhilnarang) [reply parrot/install_nvidia_drivers_post_boot.sh]: No way that could work :P

Comment (alr0cks) [reply parrot/install_nvidia_drivers_post_boot.sh]: ffff

File: parrot/setup_kde.sh (hunk 2 above)
Comment (prkprime) [reply parrot/setup_kde.sh]: Sudo is fun but it ownership goes ro root right?

**Review by prkprime — APPROVED** [review]
Approving cause nashe

**Comment by anunaym14** [comment]
Merging because @alr0cks wanted me to

**Pull request merged**